--- redland-bindings-1.0.17.1/configure.ac.orig	2014-06-22 21:07:26.186897601 +0200
+++ redland-bindings-1.0.17.1/configure.ac	2014-06-22 21:29:51.910202884 +0200
@@ -369,6 +369,9 @@
   ruby_install_dir_config=`$RUBY -rrbconfig -e "puts RbConfig::CONFIG['$ruby_install_dir_variable']"`
   ruby_header_dir_config=`$RUBY -rrbconfig -e "puts RbConfig::CONFIG['rubyhdrdir']"`
   ruby_arch_header_dir_config=`$RUBY -rrbconfig -e "puts RbConfig::CONFIG['rubyarchhdrdir']"`
+  if test -z "$ruby_arch_header_dir_config" ; then
+    ruby_arch_header_dir_config=`$RUBY -rrbconfig -e "puts RbConfig::CONFIG['rubyhdrdir']+'/'+RbConfig::CONFIG['arch']"`
+  fi
   ruby_arch_install_dir_config=`$RUBY -rrbconfig -e "puts RbConfig::CONFIG['$ruby_arch_install_dir_variable']"`
   librubyarg_shared=`$RUBY -rrbconfig -e "puts RbConfig::CONFIG['LIBRUBYARG_SHARED']"`
   changequote([, ])dnl
@@ -431,7 +431,7 @@
 	AC_MSG_ERROR(No such RUBY linking type $ruby_linking)
 	;;
   esac
-  RUBY_SO_SUFFIX=$php_linking
+  RUBY_SO_SUFFIX=$ruby_linking
   AC_MSG_CHECKING(Ruby Linking)
   AC_MSG_RESULT(compile '$RUBY_CFLAGS' link '$RUBY_LDFLAGS' suffix $RUBY_SO_SUFFIX)
 else
