--- redland-bindings-0.9.19.1/configure.ac.orig	2004-11-02 21:04:22.000000000 +0100
+++ redland-bindings-0.9.19.1/configure.ac	2005-01-09 00:14:21.283977360 +0100
@@ -190,6 +190,8 @@
   PYTHON_LIB="${python_prefix}/lib/python${PYTHON_VERSION}/site-packages"
   AC_SUBST(PYTHON_LIB)
   AC_MSG_RESULT($PYTHON_LIB)
+  PYTHON_SCRIPT="${python_prefix}/share/python${PYTHON_VERSION}/site-packages"
+  AC_SUBST(PYTHON_SCRIPT)
 
   if test `uname` = Darwin; then
     PYTHON_LDFLAGS="-Wl,-F. -Wl,-F. -bundle"
--- redland-bindings-1.0.8.1/python/Makefile.am.orig	2008-07-04 10:36:16.000000000 +0200
+++ redland-bindings-1.0.8.1/python/Makefile.am	2008-08-25 20:50:20.566120063 +0200
@@ -37,6 +37,7 @@
 SWIG_OUTPUTS=$(PYTHON_PACKAGE)_wrap.c
 
 pythondir=@PYTHON_LIB@
+pythonscriptdir=@PYTHON_SCRIPT@
 python_DATA=RDF.py Redland$(PYTHON_LIBEXT)
 
 EXTRA_DIST=README RDF.py \
@@ -66,9 +67,9 @@
 	touch $(PYTHON_PACKAGE)-stamp
 
 install-pythonDATA: $(python_DATA)
-	$(mkinstalldirs) $(DESTDIR)$(pythondir)
+	$(mkinstalldirs) $(DESTDIR)$(pythondir) $(DESTDIR)$(pythonscriptdir)
 	$(INSTALL_PROGRAM) $(PYTHON_PACKAGE)$(PYTHON_LIBEXT) $(DESTDIR)$(pythondir)/$(PYTHON_PACKAGE)$(PYTHON_LIBEXT)
-	$(INSTALL_DATA) RDF.py $(DESTDIR)$(pythondir)/RDF.py
+	$(INSTALL_DATA) RDF.py $(DESTDIR)$(pythonscriptdir)/RDF.py
 
 uninstall-pythonDATA: $(python_DATA)
 	rm -f $(DESTDIR)$(pythondir)/$(PYTHON_PACKAGE)$(PYTHON_LIBEXT) $(DESTDIR)$(pythondir)/RDF.py
