--- redland-bindings-0.9.19.1/php/redland_wrap.c.orig	2004-11-02 20:24:48.000000000 +0100
+++ redland-bindings-0.9.19.1/php/redland_wrap.c	2005-01-09 00:19:25.310758264 +0100
@@ -414,7 +414,7 @@
   p = value->ptr;
   if (type==-1) return -1;
 
-  type_name=zend_rsrc_list_get_rsrc_type(z->value.lval);
+  type_name=zend_rsrc_list_get_rsrc_type(z->value.lval TSRMLS_CC);
 
   return SWIG_ZTS_ConvertResourceData(p,type,type_name,ptr,ty TSRMLS_CC);
 }
