--- redland-bindings-1.0.5.1.orig/configure.ac	2007-03-19 11:07:04.384529443 +0200
+++ redland-bindings-1.0.5.1/configure.ac	2007-03-19 11:07:22.864944929 +0200
@@ -363,7 +363,6 @@
 
 
 
-with_ecma_cla=
 dnl Pick an ECMA CLI implementation
 AC_ARG_WITH(ecma-cli, [  --with-ecma-cli=SYSTEM  Enable ECMA CLI interface - mono, dotgnu (default=no)], ecma_cli_name="$withval", ecma_cli_name="no")
 
@@ -381,7 +380,7 @@
 AC_MSG_RESULT($result)
 
 AC_MSG_CHECKING(ECMA CLI implementations to use)
-with_ecma_cli=no
+ecma_cli=no
 CLIRUN=
 CSHARPC=
 CSHARPCFLAGS=
@@ -393,7 +392,8 @@
         CLIRUN=$MONO
         CSHARPC=$MCS
         CSHARPCFLAGS="-debug+"
-        CLI_PREFIX=`$PKG_CONFIG mono --variable=prefix`
+        CLI_PREFIX=$(cd `$PKG_CONFIG mono --variable=prefix`; pwd)
+	INSTALL_CSHARP=install-mono
         ecma_cli=Mono
       fi
       ;;
@@ -404,11 +404,13 @@
         CSHARPC=$CSCC
         CSHARPCFLAGS="-g"
         CLI_PREFIX=/usr
+	INSTALL_CSHARP=install-pnet
         ecma_cli=dotGNU
       fi
       ;;
 
     no)
+      break
       ;;
     *)
       AC_MSG_ERROR(No such ECMA CLI implementation $name)
@@ -421,8 +423,9 @@
 AC_SUBST(CSHARPCFLAGS)
 AC_SUBST(CLI_PREFIX)
 
-if test "$with_ecma_cli" != no; then
-  languages="$languages C#($ecma_cli)"
+AC_SUBST(INSTALL_CSHARP)
+if test "$ecma_cli" != no; then
+  languages="$languages csharp"
 fi
 
 
--- redland-bindings-1.0.5.1.orig/csharp/Makefile.am	2006-10-22 09:15:18.000000000 +0300
+++ redland-bindings-1.0.5.1/csharp/Makefile.am	2007-03-19 11:07:22.854944705 +0200
@@ -93,7 +93,7 @@
 	rm -f build-csharp-stamp $(CSHARP_DLL)
 
 # See this is the fake bit
-install-fakeDATA: install-csharp
+install-fakeDATA: $(INSTALL_CSHARP)
 
 
 #MONO_GACUTIL=$(CLIRUN) $(CLI_PREFIX)/gacutil.exe
@@ -124,4 +124,4 @@
 	$(MONO_GACUTIL) -i Redland.dll -f -package 1.0 -gacdir $(DESTDIR)$(CLI_PREFIX)/lib -root $(DESTDIR)$(CLI_PREFIX)/lib
 
 install-pnet: Redland.dll
-	$(PNET_GACUTIL) --install --default Redland.dll 1.0
+	$(PNET_GACUTIL) -c $(DESTDIR)$(CLI_PREFIX)/lib --install --default Redland.dll 1.0
