--- redland-bindings-1.0.11.1/configure.ac.orig	2010-09-27 03:57:15.000000000 +0200
+++ redland-bindings-1.0.11.1/configure.ac	2010-11-20 11:34:40.759094909 +0100
@@ -237,6 +237,7 @@
 AC_ARG_VAR(PYTHON_LIBEXT, [Python shared library file extension])
 AC_ARG_VAR(PYTHON_INCLUDES, [Install location of python includes])
 AC_ARG_VAR(PYTHON_LIB, [Install location of python libraries])
+AC_ARG_VAR(PYTHON_SCRIPT, [Install location of python scripts])
 
 AC_MSG_CHECKING(Enable Python API)
 if test "X$with_python" != Xno; then
@@ -297,6 +298,14 @@
     AC_MSG_RESULT($PYTHON_LIB)
   fi
 
+  AC_MSG_CHECKING(Python scripts)
+  if test "X$PYTHON_SCRIPT" != X; then
+    AC_MSG_RESULT(using user lib: $PYTHON_SCRIPT)
+  else
+    PYTHON_SCRIPT="${python_prefix}/share/python${PYTHON_VERSION}/site-packages"
+    AC_MSG_RESULT($PYTHON_SCRIPT)
+  fi
+
   AC_MSG_CHECKING(Python LDFLAGS)
   if test "X$PYTHON_LDFLAGS" != X; then
     AC_MSG_RESULT(using user ldflags: $PYTHON_LDFLAGS)
--- redland-bindings-1.0.8.1/python/Makefile.am.orig	2008-07-04 10:36:16.000000000 +0200
+++ redland-bindings-1.0.8.1/python/Makefile.am	2008-08-25 20:50:20.566120063 +0200
@@ -37,6 +37,7 @@
 SWIG_OUTPUTS=$(PYTHON_PACKAGE)_wrap.c
 
 pythondir=@PYTHON_LIB@
+pythonscriptdir=@PYTHON_SCRIPT@
 python_DATA=RDF.py Redland$(PYTHON_LIBEXT)
 
 EXTRA_DIST=README RDF.py \
@@ -66,9 +67,9 @@
 	touch $(PYTHON_PACKAGE)-stamp
 
 install-pythonDATA: $(python_DATA)
-	$(mkinstalldirs) $(DESTDIR)$(pythondir)
+	$(mkinstalldirs) $(DESTDIR)$(pythondir) $(DESTDIR)$(pythonscriptdir)
 	$(INSTALL_PROGRAM) $(PYTHON_PACKAGE)$(PYTHON_LIBEXT) $(DESTDIR)$(pythondir)/$(PYTHON_PACKAGE)$(PYTHON_LIBEXT)
-	$(INSTALL_DATA) RDF.py $(DESTDIR)$(pythondir)/RDF.py
+	$(INSTALL_DATA) RDF.py $(DESTDIR)$(pythonscriptdir)/RDF.py
 
 uninstall-pythonDATA: $(python_DATA)
 	rm -f $(DESTDIR)$(pythondir)/$(PYTHON_PACKAGE)$(PYTHON_LIBEXT) $(DESTDIR)$(pythondir)/RDF.py
