--- redland-bindings-1.0.14.1/php/redland-post.i.orig	2011-11-04 17:07:23.000000000 +0100
+++ redland-bindings-1.0.14.1/php/redland-post.i	2011-11-26 22:19:29.113100893 +0100
@@ -107,6 +107,7 @@
     return;
   
 #if PHP_MAJOR_VERSION >= 5
+  TSRMLS_FETCH();
   if(librdf_php_log_level >= LIBRDF_LOG_WARN)
     zend_throw_exception(redland_exception_ptr,
                          librdf_php_log_message,
@@ -129,7 +130,8 @@
     librdf_world_open(librdf_php_world);
 
 #if PHP_MAJOR_VERSION >= 5
-    exception_ce = zend_exception_get_default();
+    TSRMLS_FETCH();
+    exception_ce = zend_exception_get_default(TSRMLS_C);
     INIT_CLASS_ENTRY(ee_ce, "RedlandException", NULL);
     redland_exception_ptr = zend_register_internal_class_ex(&ee_ce, 
                                                             exception_ce, 
